packages/terra-form-select/src/_MenuUtil.js

```diff
@@ -22,7 +22,7 @@ class MenuUtil {
    * @return {boolean} - True if the values are equal.
    */
   static isEqual(a, b) {
-    if (a === undefined || b === undefined) {
+    if (a === undefined || a === null || b === undefined || b === null) {
```

Comment:
should we be checking for all falsy values instead?
```suggestion
    if (!a || !b) {
```

Reply:
Javascript [falsey](https://developer.mozilla.org/en-US/docs/Glossary/Falsy) includes empty string and zero which can be valid values.

When is the value null?

Reply:
Type in something that returns no results found, then press down on the keyboard. It will throw the air.

Reply:
Here's the line that returns null. https://github.com/cerner/terra-core/blob/master/packages/terra-form-select/src/_MenuUtil.js#L184

Reply:
actually, we need to leave this how it is. If we do a check on !a or !b and both a and b are empty strings '', '', then this would incorrectly return false. We essentially need to call toString on the method, which is done through null and undefined checks.